mllib/src/main/scala/org/apache/spark/ml/param/shared/sharedParams.scala

```diff
@@ -389,4 +389,21 @@ private[ml] trait HasSolver extends Params {
   /** @group getParam */
   final def getSolver: String = $(solver)
 }
+
+/**
+ * Trait for shared param aggregationDepth (default: 2).
+ */
+private[ml] trait HasAggregationDepth extends Params {
+
+  /**
+   * Param for suggested depth for treeAggregate (>= 2).
+   * @group param
```

Comment:
Could it be done in the task (https://issues.apache.org/jira/browse/SPARK-17169) ?